phoenix-pherf/pom.xml

```diff
@@ -118,6 +118,15 @@
 			<artifactId>jaxb-runtime</artifactId>
 			<scope>test</scope>
 		</dependency>
+		<dependency>
+			<groupId>junit</groupId>
+			<artifactId>junit</artifactId>
```

Comment:
Add scope `<test>`

Reply:
There's actually an existing JUnit test dependency at line 65, so I'll remove this one. I think this one came from back when my tests were in the wrong directory